vendor/assets/javascripts/jquery.jnotify.js

```diff
@@ -160,7 +160,8 @@
 
 		// remove the notification
 		this.remove = function (callback){
-			var $msg = $note.find("." + options.classMessage), $parent = $msg.parent();
+
+			var $msg = $note.find("." + options.classMessage.replace(" ",",")), $parent = $msg.parent();
```

Comment:
I'm not happy with changing vendored javascript, two questions:
1. dont we have pf alterts js built in?
2. do we need to update jnotify?

why do we need this change at all?